driver/src/price_finding/mod.rs

```diff
@@ -19,7 +19,7 @@ pub fn create_price_finder(
     } else {
         info!(
             "Using optimization price finder with the args {:?}",
-            solver_type.to_args()
+            solver_type.to_args("RESULT_FOLDER", "INPUT_FOLDER", String::from("TIME_LIMIT"))
```

Comment:
Not a big fan of these dummy placeholders. Can we expose the real values here or otherwise make the params optional?

Reply:
I delete them here, as we will get better printing in this PR: #685. It makes more sense to print them later, especially, as the time-limit is variable